test/witan/models/load_data_test.clj

```diff
@@ -0,0 +1,67 @@
+(ns witan.models.load-data-test
+  (:require [witan.models.load-data :refer :all]
+            [clojure.test :refer :all]
+            [schema.core :as s]))
+
+(defn- same-coll? [coll1 coll2]
+  (= (set coll1) (set coll2)))
+
+(deftest custom-keyword-test
+  (testing "String column names are keywordised, with '.' and ' ' replaced by '-'."
+    (is (= [:col-1 :col2 :col-3]
+           (#'witan.models.load-data/custom-keyword ["col 1" "col2" "col.3"])))))
```

Comment:
testing private functions always makes me itchy. If they are private it is b/c they are part of the implementation and might change w/o changing the interface. I'd expect to be able to change a private function without having to change a test. Same with the use of `load-csv` below.